README.md

```diff
@@ -1696,6 +1702,9 @@ IEEE Trans. Multimedia (TMM), 2020.
 - <a name="Guo20arxiv"></a>Guo, S., Kang, Z., Wang, L., Zhang, L., Chen, X., Li, S., Xu, W.,  
 *[A Noise Filter for Dynamic Vision Sensors based on Global Space and Time Information](https://arxiv.org/pdf/2004.04079)*,  
 arXiv, 2020.
+- <a name="Baldwin21arxiv"></a>Baldwin R.W., Ruixu L., Almatrafi M., Asari V., Hirakawa K.,  
```

Comment:
See the name formatting of other entries... for consistency there should be commas separating last names from given names.

Reply:
ok